src/AppBundle/Resources/views/Components/Alerts/_validation-summary.html.twig

```diff
@@ -1,15 +1,17 @@
 {% if ((formErrorMessages | length + formUncaughtErrors | length) > 0) and (error is not defined) %}
-<div id="error-summary" class="error-summary text" role="group" aria-labelledby="error-summary-heading" tabindex="-1" class="behat-region-header-errors">
-    <h1 class="heading-medium error-summary-heading" id="error-summary-heading">{{ 'summaryHeading' | trans({},'common')}}</h1>
-    <ul class="error-summary-list">
-    {% if formUncaughtErrors %}
-        {% for formError in formUncaughtErrors %}
-            <li>{{ formError.message | raw  }}</li>
+<div id="error-summary" class="govuk-error-summary" role="group" aria-labelledby="error-summary-heading" tabindex="-1">
+    <h2 class="govuk-error-summary__title" id="error-summary-heading">{{ 'summaryHeading' | trans({},'common')}}</h2>
+    <div class="govuk-error-summary__body">
+        <ul class="govuk-list govuk-error-summary__list">
+        {% if formUncaughtErrors %}
+            {% for formError in formUncaughtErrors %}
+                <li>{{ formError.message | raw  }}</li>
```

Comment:
is raw needed here? what happens if the error message references user input?

Reply:
Good point. I can't see _why_ it would be needed and I'd rather show users some unprocessed HTML than give easy access to an XSS attack. I've removed the `raw` modifier.